src/resultset/formatters/resultset_formatters.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018-2019 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Redis Labs Source Available License Agreement
+ */
+
+#pragma once
+
+#include "resultset_formatter.h"
+#include "resultset_replynop.h"
+#include "resultset_replycompact.h"
+#include "resultset_replyverbose.h"
+
+typedef enum {
+    FORMATTER_VERBOSE = 0,
+    FORMATTER_COMPACT = 1,
+    FORMATTER_NOP = 2,
```

Comment:
I think that null/unknown/nop enum values should always be 0, since it makes it a lot easier to locate errors when an enum is created but not set.

Reply:
👍